src/formaction.cpp

```diff
@@ -489,7 +489,7 @@ void FormAction::finished_qna(Operation op)
 	}
 	break;
 	case OP_INT_END_CMDLINE: {
-		f.set_focus("feeds");
+		f.set_focus(main_widget());
```

Comment:
There are two more places where we restore the focus in formaction.cpp (see occurrences of `f.set_focus(focus_before_qna);`)
I think we can just remove the `focus_before_qna` variable altogether when using this `main_widget()` implementation.

Reply:
The issue I am running into with the current implementation is that the ItemViewFormAction and HelpFormAction by default have the focus on the top-level vbox, which does not have a name.
Because of that, the `focus_before_qna` gets assigned an empty string, which we later pass as name to focus on.

If I use the new `main_widget()` in place of `focus_before_qna`, the focus seems to be restored properly (tested with stfl 0.24)